app/head.mjs

```diff
@@ -0,0 +1,19 @@
+import { getStyles }  from '@enhance/arc-plugin-styles'
+
+export default function Head () {
+  const styles = process.env.ARC_LOCAL
+    ? getStyles.linkTag()
+    : getStyles.styleTag()
+
+  return `
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <title></title>
+      ${styles}
```

Comment:
[You can just import `linkTag` and use it directly instead of having this `style` variable](https://github.com/enhance-dev/arc-plugin-styles/blob/main/test/get-styles-test.mjs#L3)